RULES.md

```diff
@@ -444,9 +450,24 @@ An example of a performance profiles plot is shown below, where we plot $\rho_{\
 
 To get a scalar score that is usable for ranking submissions, we will integrate the performance profiles $\rho_{\bar{s}}(\tau)$ of all submissions to get their benchmark score $B_{\bar{s}}$, with
 
-$$B_{\bar{s}} = \frac{1}{r_{\text{max}}} \int_{1}^{r_{\text{max}}} \rho_{\bar{s}}(\tau) \,d\tau \in [0, 1].$$
+$$B_{\bar{s}} = \frac{1}{r_{\text{max}}-1} \int_{1}^{r_{\text{max}}} \rho_{\bar{s}}(\tau) \,d\tau \in [0, 1].$$
+
+The upper integration limit will be set to $r_{\text{max}} = 4$ which also serves as the upper limit of the performance profile plot.
+This means that any submission that requires more than four times the runtime of the fastest submission will not get any credit on this workload compared to a training algorithm that is unable to successfully train within the maximum allowed runtime budget.
+The integral is normalized by the total integration area, with higher benchmark scores being better.
+
+##### Using held-out workloads in scoring
+
+For the benchmark score, we compute and integrate the performance profiles using the training times of only the fixed workloads. But we use the submission's performance on the held-out workloads to penalize submissions. Specifically, if a submission is unable to train a held-out workload, we score the submission on the corresponding fixed workload as if that submission did not reach the target. In other words, for a submission to receive a finite training time on a fixed workload, it needs to:
+
+- Reach the validation and test target on the fixed workload within the maximum runtime.
+- Reach the validation and test target fixed workload within 4x of the fastest submission.
+- Reach the validation and test target on the held-out workload (corresponding to the fixed workload) within the maximum runtime.
+- Reach the validation and test target on the held-out workload (corresponding to the fixed workload) within 4x of the fastest submission. To determine the fastest submission on a held-out workload, we only consider submissions that reached the target on the corresponding fixed workload. This protects us against extremely fast but held-out workload-specific submissions.
```

Comment:
"This protects us against extremely fast but held-out workload-specific submissions."
->
This protects us against extremely fast submissions that only work on a specific held-out workload-specific and are useless as general algorithms.